src/shared/ContentsTable/utils/adjustListIfUpDir.tsx

```diff
@@ -15,13 +15,13 @@ type TreePath = {
   }
 }
 
-type Row = {
+export type Row = {
   name: ReactNode
-  lines: string
-  hits: string
-  misses: string
-  partials: string
-  coverage: ReactNode
+  lines?: string | number | null
+  hits?: string | number | null
+  misses?: string | number | null
+  partials?: string | number | null
+  coverage?: ReactNode | number | null
```

Comment:
You shouldn't need to add in `number` and `null` here for `coverage` as `ReactNode` should already account for those types.

Reply:
@rohitvinnakota-codecov comparing this to my stuff in the pull file explorer, it seems these are required because your zod schema states that lines, hits, misses, partials, and coverage are nullable. The graphql schema states these are all non-null and having looked on the API side, they seem to fall back to 0. Do you have an example of a PathContents query where these are null? If so I'll need to adjust my schema.

Reply:
You're right about the zod schema. updated.